packages/yourloops/components/dashboard-widgets/patient-info-widget.tsx

```diff
@@ -77,12 +81,22 @@ function PatientInfoWidget(props: PatientInfoWidgetProps): JSX.Element {
   const { t } = useTranslation("yourloops");
   const trNA = t("N/A");
   const authHook = useAuth();
+  const notificationHook = useNotification();
+  const teamHook = useTeam();
   const [showInviteRemoteMonitoringDialog, setShowInviteRemoteMonitoringDialog] = useState(false);
+  const [showConfirmDialog, setShowConfirmDialog] = useState(false);
+  const [confirmDialogActionInProgress, setConfirmDialogActionInProgress] = useState(false);
   const hbA1c: Settings["a1c"] = patient.settings.a1c
     ? { value: patient.settings.a1c.value, date: moment.utc(patient.settings.a1c.date).format("L") }
     : undefined;
   const birthdate = moment.utc(patient.profile.birthdate).format("L");
 
+  const isLoggedInUserHcpAdmin = () => {
+    return authHook.user?.isUserHcp() && teamHook.getRemoteMonitoringTeams().find(team => team.members.find(member => member.role === TeamMemberRole.admin && member.user.userid === authHook.user?.userid)) !== undefined;
```

Comment:
We check that the user is an admin in one of the remote monitoring team he's in, but not an admin in the team the patient is in.